addons/knobs/package.json

```diff
@@ -29,14 +29,14 @@
     "util-deprecate": "^1.0.2"
   },
   "devDependencies": {
-    "@types/node": "^7.0.12",
+    "@types/node": "~6.0.60",
```

Comment:
In that specific case the TS definition was already there. I downgraded it because typescript has issues with monorepos and hoisting  and wasn't being able to compile.
There are several github issues with no real fix for the problem (that I could make work) other than making the versions match.